### ISSUE TITLE
mAP and loss value is fluctuated

### ISSUE THREAD

**Issue by khgapa** [eriklindernoren/PyTorch-YOLOv3#231]
I trained with my custom dataset. mAP and loss values were logged. 
I think these values don't increase and decrease smoothly. 
What do you think about that?
Could you help me this problem?

epoch = 305   mAP = 0.5873778260528077  Loss = 0.3016989231109619
epoch = 310   mAP = 0.39661738345878805  Loss = 1.5647668838500977
epoch = 315   mAP = 0.622580115604962  Loss = 0.5903745889663696
epoch = 320   mAP = 0.6027897803611924  Loss = 1.0634759664535522
epoch = 325   mAP = 0.5789129436417217  Loss = 0.29768702387809753
epoch = 330   mAP = 0.6418522475763313  Loss = 0.32770904898643494
epoch = 335   mAP = 0.6045293065216031  Loss = 0.21479982137680054
epoch = 340   mAP = 0.6677603465781639  Loss = 0.2527669072151184
epoch = 345   mAP = 0.6768549406998039  Loss = 1.28099524974823
epoch = 350   mAP = 0.41159787778366286  Loss = 0.814625084400177
epoch = 355   mAP = 0.6141030938710429  Loss = 0.31631988286972046
epoch = 360   mAP = 0.6797186677396986  Loss = 0.9694105386734009
epoch = 365   mAP = 0.5622327566398808  Loss = 3.1963677406311035
epoch = 370   mAP = 0.6519483495430556  Loss = 0.7326493263244629
epoch = 375   mAP = 0.3342195464753914  Loss = 0.37378454208374023
epoch = 380   mAP = 0.6114800300699555  Loss = 0.9522882699966431
epoch = 385   mAP = 0.5513716505300894  Loss = 0.9601264595985413
epoch = 390   mAP = 0.6965701912435156  Loss = 0.33222734928131104
epoch = 395   mAP = 0.718384561040636  Loss = 0.06073024123907089
epoch = 400   mAP = 0.7113701444534444  Loss = 0.14153385162353516
epoch = 405   mAP = 0.5704689228887405  Loss = 0.31103625893592834
epoch = 410   mAP = 0.7183532124216718  Loss = 0.10084554553031921
epoch = 415   mAP = 0.6087351357798703  Loss = 0.1788690835237503
epoch = 420   mAP = 0.5325617060790786  Loss = 0.18996770679950714
epoch = 425   mAP = 0.2702706772012675  Loss = 0.5356136560440063
epoch = 430   mAP = 0.3063981381033922  Loss = 11.564997673034668
epoch = 435   mAP = 0.6501270053995098  Loss = 0.41580986976623535
epoch = 440   mAP = 0.613765044416453  Loss = 0.220064178109169
epoch = 445   mAP = 0.35998462789998725  Loss = 1.12434983253479
epoch = 450   mAP = 0.6644974072507158  Loss = 0.43725281953811646
epoch = 455   mAP = 0.6451055902966153  Loss = 0.2519175708293915
epoch = 460   mAP = 0.6788040014395851  Loss = 0.09991863369941711
epoch = 465   mAP = 0.6282575248239688  Loss = 0.11920318752527237
epoch = 470   mAP = 0.6870619673303658  Loss = 0.5322765707969666
epoch = 475   mAP = 0.573181278599501  Loss = 0.17670898139476776
epoch = 480   mAP = 0.3800835659769526  Loss = 0.0952586755156517
epoch = 485   mAP = 0.6895224462978885  Loss = 2.4513444900512695
epoch = 490   mAP = 0.6526468006948943  Loss = 1.381237506866455
epoch = 495   mAP = 0.3946252575678779  Loss = 0.36981940269470215
epoch = 500   mAP = 0.7023063158182086  Loss = 0.1646282970905304

**Comment by abhiksark** [comment]
What was the Batch Size?

**Comment by khgapa** [comment]
@abhiksark 10~12 images

**Comment by abhiksark** [comment]
Small Batch Size Might have been an issue here

**Comment by khgapa** [comment]
@abhiksark  Thank you. I need to  solve the multiple gpus problem to increase batch size.

**Comment by wurongyuan** [comment]
> @abhiksark Thank you. I need to solve the multiple gpus problem to increase batch size.

how about it after increasing the batch size?

**Comment by Flova** [comment]
You could also increase the gradient accumulations
